helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/InstancesAccessor.java

```diff
@@ -198,7 +194,7 @@ public Response instancesOperations(@PathParam("clusterId") String clusterId,
           break;
         case stoppable:
           return batchGetStoppableInstances(clusterId, node, skipZKRead, continueOnFailures,
-              skipHealthCheckCategorySet);
```

Comment:
comment about default value.

Reply:
Also add sanity check when random and zone order is set at same time.

Reply:
Thanks for the review. Issues are fixed.